package.json

```diff
@@ -15,6 +15,8 @@
   "maintainer": "Alex Hsu <alex.hsu@plot.ly>",
   "license": "MIT",
   "dependencies": {
+    "16": "0.0.2",
+    "npm": "^8.15.0",
```

Comment:
What's 16? Anyway I wouldn't list npm/node in dependencies, but if you want to list their recommended versions in `CONTRIBUTING.md` that could be worthwhile.

Reply:
Not sure how that got there. I moved removed dependencies from `package.json` and listed recommended versions in `CONTRIBUTING.md`